<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Xlive</class>
 <widget class="QMainWindow" name="Xlive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2220</width>
    <height>1194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>2220</width>
    <height>1194</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2220</width>
    <height>1194</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XLive NSLS-II</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>1641</width>
      <height>961</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab_beamline_setup_manager">
     <attribute name="title">
      <string>Beamline setup</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1631</width>
        <height>931</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_beamline_setup">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_sdd_manager">
     <attribute name="title">
      <string>Silicon Drift Detector Setup</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1631</width>
        <height>931</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_sdd_manager">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_trajectory_manager">
     <attribute name="title">
      <string>Trajectory Setup</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1631</width>
        <height>931</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_trajectory_manager">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Run</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1631</width>
        <height>931</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_run">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Run Batch</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1631</width>
        <height>931</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_batch">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>New Batch Mode</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1631</width>
        <height>921</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_batch_new"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Processing</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1631</width>
        <height>931</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_processing">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_terminal">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>190</y>
      <width>541</width>
      <height>951</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1120</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_70">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>State:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="push_re_abort">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>1120</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Abort scan</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>1120</y>
      <width>1301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_59">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>170</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Terminal window</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layout_general_info">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>1131</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layout_beamline_status"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelct_new_trajectory">
   <property name="text">
    <string>Select new trajectory...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
